sites/svelte.dev/scripts/type-gen/index.js

```diff
@@ -0,0 +1,390 @@
+
+	modules.push({
+		name: 'svelte',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+{
+	const module = bundled_types.get('svelte/compiler');
+
+	if (!module) throw new Error('Could not find svelte/compiler');
+
+	modules.push({
+		name: 'svelte/compiler',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+{
+	const module = bundled_types.get('svelte/action');
+
+	if (!module) throw new Error('Could not find svelte/action');
+
+	modules.push({
+		name: 'svelte/action',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+{
+	const module = bundled_types.get('svelte/animate');
+
+	if (!module) throw new Error('Could not find svelte/animate');
+
+	modules.push({
+		name: 'svelte/animate',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+{
+	const module = bundled_types.get('svelte/easing');
+
+	if (!module) throw new Error('Could not find svelte/easing');
+
+	modules.push({
+		name: 'svelte/easing',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+{
+	const module = bundled_types.get('svelte/motion');
+
+	if (!module) throw new Error('Could not find svelte/motion');
+
+	modules.push({
+		name: 'svelte/motion',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+{
+	const module = bundled_types.get('svelte/store');
+
+	if (!module) throw new Error('Could not find svelte/store');
+
+	modules.push({
+		name: 'svelte/store',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+{
+	const module = bundled_types.get('svelte/transition');
+
+	if (!module) throw new Error('Could not find svelte/transition');
+
+	modules.push({
+		name: 'svelte/transition',
+		comment: '',
+		...get_types(module.code, module.ts_source_file.statements)
+	});
+}
+
+// const dir = fileURLToPath(
+// 	new URL('../../../../packages/kit/types/synthetic', import.meta.url).href
+// );
+// for (const file of fs.readdirSync(dir)) {
+// 	if (!file.endsWith('.md')) continue;
+
+// 	const comment = strip_origin(read_d_ts_file(`${dir}/${file}`));
+
+// 	modules.push({
+// 		name: file.replace(/\+/g, '/').slice(0, -3),
+// 		comment,
+// 		exports: [],
+// 		types: [],
+// 		exempt: true
+// 	});
+// }
+
+// {
+// 	const code = read_d_ts_file('types/ambient.d.ts');
+// 	const node = ts.createSourceFile('ambient.d.ts', code, ts.ScriptTarget.Latest, true);
+
+// 	for (const statement of node.statements) {
+// 		if (ts.isModuleDeclaration(statement)) {
+// 			// @ts-ignore
+// 			const name = statement.name.text || statement.name.escapedText;
+
+// 			// @ts-ignore
+// 			const comment = strip_origin(statement.jsDoc?.[0].comment ?? '');
+
+// 			modules.push({
+// 				name,
+// 				comment,
+// 				// @ts-ignore
+// 				...get_types(code, statement.body?.statements)
+// 			});
+// 		}
+// 	}
+// }
```

Comment:
What's this commented-out code about?

Reply:
Copied code from sveltekit's typegen script. I'm still keeping em around, cuz these parts can be introduced in our site once we expose different kind of modules

Reply:
Actually nvm, I removed it. Can always copy it from git